sentinel-core/src/main/java/com/alibaba/csp/sentinel/config/SentinelConfig.java

```diff
@@ -98,6 +103,21 @@ private static void resolveAppType() {
         }
     }
 
+    private static void resolveMaxContextNameSize() {
+        try {
+            String size = getConfig(MAX_CONTEXT_NAME_SIZE);
+            if (size == null) {
+                return;
+            }
+            maxContextNameSize = Integer.parseInt(size);
+            if (maxContextNameSize < 0) {
+                maxContextNameSize = DEFAULT_MAX_CONTEXT_NAME_SIZE;
+            }
+        } catch (Exception ex) {
+            maxContextNameSize = DEFAULT_MAX_CONTEXT_NAME_SIZE;
+        }
```

Comment:
log an error message here?

Reply:
In fact, I wrote it with reference to the `resolveAppType` method. There is no warning message in the `resolveAppType` method